obal/data/module_utils/obal.py

```diff
@@ -16,15 +32,25 @@ def get_changelog_evr(specfile):
 
 def get_specfile_evr(specfile):
     """get the EVR from the source header of the specfile"""
-    return subprocess.check_output([
-        'rpmspec',
-        '--query',
-        '--queryformat',
-        '%{evr}',
-        '--undefine',
-        'dist',
-        '--undefine',
-        'foremandist',
-        '--srpm',
-        specfile
-    ])
+    return specfile_macro_lookup(specfile, '%{evr}')
+
+def get_specfile_name(specfile):
+    """get the name from the specfile"""
+    return specfile_macro_lookup(specfile, '%{name}')
+
+def get_whitelist_status(build_command, tag, package):
+    """
+    Get whitelist status of a given package within a tag.
+
+    Return `True` if the package is whitelisted, `False` otherwise.
+    """
+    cmd = [
+        build_command,
+        'list-pkgs',
+        '--tag',
+        tag,
+        '--package',
+        package,
+        '--quiet'
+    ]
+    return not bool(subprocess.call(cmd))
```

Comment:
Clever, but I'd prefer
```python
retcode = subprocess.call(cmd)
return retcode == 0
```
as I think this is more readable?

Reply:
Your suggestion is clearer.